bundler.d/fog.rb

```diff
@@ -1,4 +1,4 @@
 group :fog do
-  gem 'fog', '1.30.0', :require => false
-  gem 'fog-core', '1.29.0'
+  gem 'fog', '1.32.0', :require => false
+  gem 'fog-core', '1.32.0'
```

Comment:
I take it we can only do this finally because [json in excon](https://github.com/excon/excon/blob/master/excon.gemspec#L65) is a development dependency, did this change lately?